book/tests.py

```diff
@@ -1,3 +1,141 @@
+            id                = 1, 
+            social_id         = "01058974859",
+            nickname          = "nickname",
+            mobile            = "01058974859",
+            password          = "password",
+            birth             = 900922,
+            gender            = 1,
+            email             = "hyeseong43@gmail.com",
+            profile_image_url = "profile_image_url",
+            library_image_url = "library_image_url",
+            usertype          = usertype,
+             )
+
+        book = Book.objects.create(
+            id               = '1',
+            title            = 'title',
+            summary          = 'summary',
+            translator       = 'translator',
+            sub_title        = 'sub_title',
+            description      = 'description',
+            page             = 1,
+            capacity         = 1,
+            pub_date         = "1990-09-22",
+            launched_date    = "1990-09-22",
+            contents         = 'contents',
+            publisher_review = 'publisher_review',
+            image_url        = 'image_url',
+            purchase_url     = 'purchase_url',
+            author           = author,
+            category         = category,
+            publisher        = publisher,
+                
+        )
+        book.shelf.add(shelf)
+
+    def tearDown(self):
+        User.objects.all().delete()
+        UserType.objects.all().delete()
+        Book.objects.all().delete()
+        Publisher.objects.all().delete()
+        Subcategory.objects.all().delete()
+        Category.objects.all().delete()
+        Author.objects.all().delete()
+        Library.objects.all().delete()
+        Review.objects.all().delete()
+        ReviewLike.objects.all().delete()
+        Author.objects.all().delete()
+
+    def test_booklist_get_success(self):
+        client   = Client()
+        response = client.get('/book?category_id=1')
+        book     = Book.objects.get(id=1)
+        self.assertEqual(response.status_code, 200)
+        
+        self.assertEqual(response.json(), {'bookData': {
+                                              "slider"            : 
+                                                                    [{
+                                                                    'id'           : 1,
+                                                                    'category.id'  : 1,
+                                                                    'category.name': '일반소설',
+                                                                    'book_id'      : 1,
+                                                                    'bookTitle'    :'title',
+                                                                    'bookCoverImg' :'image_url',
+                                                                    'bookAuthor'   : 'name',
+                                                                }],
```

Comment:
```suggestion
        self.assertEqual(response.json(), {
            'bookData': {
                "slider"            : 
                                                                    [{
                                                                    'id'           : 1,
                                                                    'category.id'  : 1,
                                                                    'category.name': '일반소설',
                                                                    'book_id'      : 1,
                                                                    'bookTitle'    :'title',
                                                                    'bookCoverImg' :'image_url',
                                                                    'bookAuthor'   : 'name',
                                                                }],
```

코드 앞 부분에 여백이 너무 길어서 가독성을 해칠 수 있습니다 :)
앞에 공백 줄여주세요~

Reply:
알겠숨돠~~~~